torch/testing/_internal/common_methods_invocations.py

```diff
@@ -575,26 +606,24 @@ def sample_inputs_div(self, device, dtype, requires_grad, rounding_mode=None):
     ]
 
 def sample_inputs_stack(op_info, device, dtype, requires_grad):
-    return (SampleInput((make_tensor((S, S), device, dtype,
-                                     low=None, high=None,
-                                     requires_grad=requires_grad),
-                        make_tensor((S, S), device, dtype,
-                                    low=None, high=None,
-                                    requires_grad=requires_grad),
-                        make_tensor((S, S), device, dtype,
-                                    low=None, high=None,
-                                    requires_grad=requires_grad)), kwargs=dict(idx=0)),)
+    tensors = [make_tensor((S, S), device, dtype,
```

Comment:
For now, TensorList inputs must be List[Tensor]. @heitorschueroff find out why Tuple[Tensor] is not supported here.